notebooks/python/1.1. BigQuery Storage & Spark DataFrames - Python.ipynb

```diff
@@ -0,0 +1,374 @@
+      "text/html": [
+       "<table border='1'>\n",
+       "<tr><th>title</th><th>wiki</th><th>views</th></tr>\n",
+       "<tr><td>2020_Democratic_P...</td><td>en</td><td>3242</td></tr>\n",
+       "<tr><td>Eurovision_Song_C...</td><td>en</td><td>2368</td></tr>\n",
+       "<tr><td>Colin_McRae</td><td>en</td><td>2360</td></tr>\n",
+       "<tr><td>Donald_trump</td><td>en</td><td>2223</td></tr>\n",
+       "<tr><td>Comparison_of_onl...</td><td>en</td><td>1398</td></tr>\n",
+       "<tr><td>Coronavirus</td><td>en</td><td>1872</td></tr>\n",
+       "<tr><td>-</td><td>en</td><td>136620</td></tr>\n",
+       "<tr><td>Bombshell_(2019_f...</td><td>en</td><td>1084</td></tr>\n",
+       "<tr><td>Brooklyn</td><td>en</td><td>1946</td></tr>\n",
+       "<tr><td>2019–20_coronavir...</td><td>en</td><td>8313</td></tr>\n",
+       "<tr><td>2019–20_Wuhan_cor...</td><td>en</td><td>1084</td></tr>\n",
+       "<tr><td>Apple_Network_Server</td><td>en</td><td>3524</td></tr>\n",
+       "<tr><td>Catholic_moral_th...</td><td>en</td><td>1328</td></tr>\n",
+       "<tr><td>Bernie_Sanders</td><td>en</td><td>1297</td></tr>\n",
+       "<tr><td>2019–20_coronavir...</td><td>en</td><td>1968</td></tr>\n",
+       "<tr><td>Brooklyn</td><td>en</td><td>1139</td></tr>\n",
+       "<tr><td>Charlie&#x27;s_Angels_...</td><td>en</td><td>1006</td></tr>\n",
+       "<tr><td>Corrupted_Blood_i...</td><td>en</td><td>1511</td></tr>\n",
+       "<tr><td>Donald_trump</td><td>en</td><td>1526</td></tr>\n",
+       "<tr><td>Coronavirus_disea...</td><td>en</td><td>1405</td></tr>\n",
+       "</table>\n",
+       "only showing top 20 rows\n"
+      ],
+      "text/plain": [
+       "+--------------------+----+------+\n",
+       "|               title|wiki| views|\n",
+       "+--------------------+----+------+\n",
+       "|2020_Democratic_P...|  en|  3242|\n",
+       "|Eurovision_Song_C...|  en|  2368|\n",
+       "|         Colin_McRae|  en|  2360|\n",
+       "|        Donald_trump|  en|  2223|\n",
+       "|Comparison_of_onl...|  en|  1398|\n",
+       "|         Coronavirus|  en|  1872|\n",
+       "|                   -|  en|136620|\n",
+       "|Bombshell_(2019_f...|  en|  1084|\n",
+       "|            Brooklyn|  en|  1946|\n",
+       "|2019–20_coronavir...|  en|  8313|\n",
+       "|2019–20_Wuhan_cor...|  en|  1084|\n",
+       "|Apple_Network_Server|  en|  3524|\n",
+       "|Catholic_moral_th...|  en|  1328|\n",
+       "|      Bernie_Sanders|  en|  1297|\n",
+       "|2019–20_coronavir...|  en|  1968|\n",
+       "|            Brooklyn|  en|  1139|\n",
+       "|Charlie's_Angels_...|  en|  1006|\n",
+       "|Corrupted_Blood_i...|  en|  1511|\n",
+       "|        Donald_trump|  en|  1526|\n",
+       "|Coronavirus_disea...|  en|  1405|\n",
+       "+--------------------+----+------+\n",
+       "only showing top 20 rows"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df1 = df \\\n",
```

Comment:
Nit: `df`, `df1`, and `df2` aren't very good variable names. The last cell references `df2`, and it's not immediately clear which table that represents.